backbone/src/main/java/org/researchstack/backbone/ui/step/layout/SurveyStepLayout.java

```diff
@@ -251,7 +251,7 @@ protected void onNextClicked()
     {
         BodyAnswer bodyAnswer = stepBody.getBodyAnswerState();
 
-        if(bodyAnswer == null || ! bodyAnswer.isValid())
+        if ((bodyAnswer == null || ! bodyAnswer.isValid()) && !readOnlyMode)
```

Comment:
Indentation and spaces on this clause are pretty inconsistent

Reply:
inherited 😞